pkg/apis/crunchydata.com/v1/cluster.go

```diff
@@ -157,6 +157,36 @@ type PgclusterSpec struct {
 	Tolerations []v1.Toleration `json:"tolerations"`
 
 	PMM PMMSpec `json:"pmm"`
+
+	PGReplicas *PGReplicas `json:"pgReplicas"`
+}
+
+type PGReplicas struct {
+	HotStandby HotStandby `json:"hotStandby"`
+}
+
+type HotStandby struct {
+	Size              int               `json:"size"`
+	Resources         Resources         `json:"resources"`
+	Storage           *PgStorageSpec    `json:"storage"`
+	Labels            map[string]string `json:"labels"`
+	Annotations       map[string]string `json:"annotations"`
+	Affinity          *v1.NodeAffinity  `json:"affinity"`
+	EnableSyncStandby *bool             `json:"enableSyncStandby"`
+	Expose            Expose            `json:"expose"`
+}
+
+type Expose struct {
+	Enabled                  bool              `json:"enabled"`
+	ServiceType              v1.ServiceType    `json:"serviceType"`
+	LoadBalancerSourceRanges []string          `json:"loadBalancerSourceRanges"`
+	Annotations              map[string]string `json:"annotations"`
```

Comment:
annotations do not work

Reply:
fixed in https://github.com/percona/percona-postgresql-operator/pull/50/commits/b8072fb98fc6630be47a1f23d684c671d360cfe8